x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/cti/indicator_details.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import {
+  INDICATOR_DATASET,
+  INDICATOR_MATCHED_TYPE,
+  INDICATOR_PROVIDER,
+  INDICATOR_REFERENCE,
+} from '../../../../../../../common/cti/constants';
+import { DraggableBadge } from '../../../../../../common/components/draggables';
+import { FormattedFieldValue } from '../formatted_field';
+import { HorizontalSpacer } from './helpers';
+
+interface IndicatorDetailsProps {
+  contextId: string;
+  eventId: string;
+  indicatorDataset: string | undefined;
+  indicatorProvider: string | undefined;
+  indicatorReference: string | undefined;
+  indicatorType: string | undefined;
+}
+
+export const IndicatorDetails: React.FC<IndicatorDetailsProps> = ({
+  contextId,
+  eventId,
+  indicatorDataset,
+  indicatorProvider,
+  indicatorReference,
+  indicatorType,
+}) => (
+  <EuiFlexGroup
+    alignItems="flexStart"
+    data-test-subj="threat-match-indicator-details"
+    direction="row"
+    justifyContent="center"
+    gutterSize="none"
+    wrap
+  >
+    {indicatorType && (
+      <EuiFlexItem grow={false}>
+        <DraggableBadge
+          contextId={contextId}
+          data-test-subj="threat-match-indicator-details-indicator-type"
+          eventId={eventId}
+          field={INDICATOR_MATCHED_TYPE}
+          value={indicatorType}
+        />
+      </EuiFlexItem>
+    )}
+    {indicatorDataset && (
+      <>
+        <EuiFlexItem grow={false}>
+          <HorizontalSpacer>
+            <FormattedMessage
```

Comment:
This is just a react wrapper around the same logic. Since I don't expect/intend for these strings to be shared, they can just live in the component itself (for now).